helix-core/src/main/java/org/apache/helix/manager/zk/ZkCacheBaseDataAccessor.java

```diff
@@ -232,7 +232,7 @@ public boolean create(String path, T data, int options) {
       try {
         cache.lockWrite();
         ZkBaseDataAccessor<T>.AccessResult result =
-            _baseAccessor.doCreate(serverPath, data, options);
+            _baseAccessor.doCreate(_zkclient, serverPath, data, options);
```

Comment:
If we keep the old public API instead of removing, we shouldn't need to change this at all. In other words, we shouldn't see any diffs in CacheBaseDataAccessor.